### PR TITLE
Improve images and examples

### PR DESCRIPTION
**Pull request by NipunaRanasinghe**
## Purpose
$subject.

Related https://github.com/ballerina-platform/ballerina-library/issues/6238

## Checklist
- [ ] Linked to an issue
- [ ] Updated the changelog
- [ ] Added tests
- [ ] Updated the spec
- [ ] Checked native-image compatibility

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Examples
+
+The `ballerinax/asana` connector provides practical examples illustrating usage in various scenarios. Explore these [examples](https://github.com/ballerina-platform/module-ballerinax-asana/tree/master/examples), covering use cases like cache management, session management, and rate limiting.
```
Comment (Nuvindu) [inline examples/README.md]:
```suggestion
The `ballerinax/asana` connector provides practical examples illustrating usage in various scenarios. Explore these [examples](https://github.com/ballerina-platform/module-ballerinax-asana/tree/main/examples), covering use cases like cache management, session management, and rate limiting.
```

File: examples/README.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+**Warning**: Due to the absence of support for reading local repositories for single Ballerina files, the Bala of the module is manually written to the central repository as a workaround. Consequently, the bash script may modify your local Ballerina repositories.
+
+Execute the following commands to build all the examples against the changes you have made to the module locally:
+
+* To build all the examples:
+
+    ```bash
+    ./build.sh build
+    ```
```
Comment (ayeshLK) [inline examples/README.md:47]: As per this instruction we have to commit a `build.sh` file with required functionality. Is this included in the repository ?

Comment (NipunaRanasinghe) [reply examples/README.md:47]: You are correct. This was missing and the required files are added with https://github.com/ballerina-platform/module-ballerinax-asana/pull/7/commits/26e231bb6bb68556de771513f7dff4abf1233f69

**Review by ayeshLK — APPROVED** [review]
LGTM

**Pull request merged**